fe/fe-core/src/main/java/org/apache/doris/planner/PlanNode.java

```diff
@@ -984,4 +984,17 @@ public String toString() {
         sb.append("\n").append(getNodeExplainString("", TExplainLevel.BRIEF));
         return sb.toString();
     }
+
+    /**
+     * Used to append some common explains to output
+     */
+    protected void appendCommonExplainString(String detailPrefix, StringBuilder output) {
```

Comment:
It seems that there is already a getExplainString in the plan node for common string output.